docs/monitoring/monitoring.md

```diff
@@ -1,241 +1,237 @@
-   Displays the data distribution type for the table. Possible values are PARTITION, PARTITION_PERSISTENT, PARTITION_REDUNDANT, PARTITION_OVERFLOW, REPLICATE, REPLICATE_PERSISTENT, REPLICATE_OVERFLOW etc.
+<a id="member"></a>
+### Members
+In the **Members** section,  you can view, in a tabular format, the details of each locator, server, and lead member within a cluster. The details are automatically refreshed after every 5 seconds.
+![Members](../Images/Dashboard-MembersList.png)
 
-* **Row Count**</br>
-   Displays the row count, which is the number of records present in the data table.
+This table provides member details in the following columns:
 
-* **Memory Size**</br>
-   Displays the heap memory used by data table to store its data/records.
 
-* **Total Size**</br>
-   Displays the collective physical memory and disk overflow space used by the data table to store its data/records.
-<!--
- **BUCKETS**</br>
-   Displays the total number of buckets in the data table.
--->
+| Column | Description |
+|--------|--------
+|  **Status**     |  Displays the status of the members, which can be either [Running or Stopped](#statusofmembers).    |
+|   **Member**       |   Displays a brief description of the member. Click the link in the column to view the Member Details page where the usage trends and statistics of the members are shown along with the [Member Logs](#memberlogs). Click the drop-down arrow to find information such as the IP address of the host, the current working directory, and the Process ID number.  |
+|     **Type**   |     Displays the type of the member. The type can be LEAD, LOCATOR, or DATA SERVER. The name of the active lead member is displayed in bold letters. |
+|     **CPU Usage**        |      Displays the CPU utilized by the member's host.  |
+|     **Memory Usage**       |      Displays the collective Heap and Off-Heap memory utilization of a cluster member. |
+|  **Heap Memory**       | Displays the member's utilized Heap memory versus total Heap memory. Click the down arrow in this column to view the detailed distribution of the member's Heap Memory for storage, execution, and JVM. |
+|    **Off-Heap Memory**     |     Displays the member's used Off-Heap memory and total Off-Heap memory. Click the down arrow in this column to view the detailed distribution of the member's Off-Heap memory for storage and execution.  |
 
-![Dashboard](../Images/monitoring_external_table.png)
-<a id="external-table"></a>
-### External Table Statistics
+<a id="statusofmembers"></a>
 
-* **Tables Count**</br>
-	Displays the total number of external tables present in the cluster. The tooltip displays the total number of external tables.
+| Status | Description |
+|--------|--------|
+|![Running](../Images/running-status.png)|Member is running.|
+|![Stopped](../Images/stopped-status.png)|Member has stopped or is unavailable.|
 
-* **Name**</br>
-	Displays the name of the external table.
+#### Member Details View
```

Comment:
Please shift the Member Details View after Cluster Page and before the Jobs Page.
Similarly, also add _Member Details View_ link like you have added for other UI pages.

Reply:
Also mention that how user can access it on UI. 
I mean mention that, user needs click on link given in Members tabular list to view members detail view of particular member.

Reply:
Member Details View image is cut logs section. Can you replace it with full image ?